src/models/m_identity.erl

```diff
@@ -542,22 +543,34 @@ get_rsc(Id, Type, Context) ->
     z_db:assoc_row("select * from identity where rsc_id = $1 and type = $2", [Id, Type], Context).
 
 
-%% @doc Hash a password, using sha1 and a salt
+%% @doc Hash a password, using bcrypt
 %% @spec hash(Password) -> tuple()
 hash(Pw) ->
-    Salt = binary_to_list(z_ids:id(10)),
-    Hash = crypto:hash(sha, [Salt,Pw]),
-    {hash, Salt, Hash}.
+    {bcrypt, erlpass:hash(Pw)}.
 
 
+%% @doc Hash a password, with bcrypt
+%% @spec bcrypt_hash(Password) -> tuple
+
 %% @doc Compare if a password is the same as a hash.
 %% @spec hash_is_equal(Password, Hash) -> bool()
+hash_is_equal(Pw, {bcrypt, Hash}) ->
```

Comment:
Thanks for finding this. 

No, I first embedded simple unit tests in `m_identity.erl` only to find out these kind of tests are not run. Then I added a new test module named `m_identity_db_tests.erl` to `src/models/tests`  because I also wanted to test the hash upgrading mechanism only to find out the tests implemented there are skipped.